.github/workflows/deploy.yml

```diff
@@ -7,7 +7,7 @@ on:
 jobs:
   build-and-deploy:
     runs-on: ubuntu-latest
-    if: github.repository == 'djangogirls/tutorial'
+    if: github.repository == 'CERN/django-girls-tutorial'
```

Comment:
Oops, sorry. I forgot that any changes to the branch would also be included in the Pull Request. Obviously, we can't merge that change or the action would break for this upstream repo here ([djangogirls/tutorial](https://github.com/DjangoGirls/tutorial)).

Reply:
Also didn't see that one coming, I'll generate from a different branch.